src/stacks-hierarchy/Root_TabNavigatorStack/TabNav_ProfileStack/Profile_RootScreen.tsx

```diff
@@ -272,93 +215,44 @@ export const Profile_RootScreen = ({navigation}: ProfileProps) => {
             )}
           </Section>
 
-          <ContentHeading text={t(ProfileTexts.sections.settings.heading)} />
           <Section>
```

Comment:
Margins are slightly off here, giving the settings section too little top padding. Probably makes sense to solve it like it's done in figma, and use `rowGap: medium` in a sub-view.

![Screenshot 2025-05-19 at 14 42 10](https://github.com/user-attachments/assets/c4deaffc-804e-444b-be33-82ead9def854)

Reply:
![Screenshot_1747662894](https://github.com/user-attachments/assets/a603eebe-6603-4dd7-b727-a929b7779178) Fixed, now it uses medium gap.

Reply:
Now it looks like everything has medium padding, which is not the case in Figma. I was thinking something like this, which would only apply medium gap to the top section

```tsx
<View style={styles.mediumGap}>
  <Section>
    {authenticationType === 'phone' && (
      <GenericSectionItem>
        <ThemeText style={style.customerNumberHeading}>
          {t(ProfileTexts.sections.account.infoItems.phoneNumber)}
        </ThemeText>
        {phoneNumber && (
  // ...

  <Section>
    <LinkSectionItem
      text={t(ProfileTexts.sections.settings.heading)}
      onPress={() => navigation.navigate('Profile_SettingsScreen')}
      testID="settingsButton"
    />
  </Section>
</View>